### ISSUE TITLE
Consider removing blas_scipy.py

### ISSUE THREAD

**Issue by ricardoV94** [aesara-devs/aesara#988]
This file includes an alternative `Ger``Op` that goes through scipy when in Python mode.

This seems unnecessary, We have many more such routines implemented in `blas.py`, and it seems we could just call the scipy routines directly in the python perform mode there. I don't know if this separation was because scipy was original an optional dependency, or if the blas module is not always available in scipy, but even if that's the case it seems simpler to just combine the scipy and numpy fallback in the same `Op`.

https://github.com/aesara-devs/aesara/blob/main/aesara/tensor/blas_scipy.py

**Comment by brandonwillard** [comment]
> I don't know if this separation was because scipy was original an optional dependency, or if the blas module is not always available in scipy, but even if that's the case it seems simpler to just combine the scipy and numpy fallback in the same `Op`.

Yes, I believe it was because SciPy was optional, and, yes, there's no need for separate modules.
